cookiecutter.json

```diff
@@ -2,13 +2,14 @@
   "author_given_names": "Eva Lu",
   "author_family_names": "Ator",
   "author_email": "eva.lu.ator@ucl.ac.uk",
-  "github_username": "{{ cookiecutter.author_given_names.lower().replace(' ', '-') }}-{{ cookiecutter.author_family_names.lower().replace(' ', '-') }}",
   "project_name": "Python Template",
   "project_slug": "{{ cookiecutter.project_name.lower().replace(' ', '-').replace('_', '-') }}",
   "package_name": "{{ cookiecutter.project_slug.replace('-', '_') }}",
   "project_short_description": "A cookiecutter template with ARC recommendations.",
-  "funder": "JBFC: The Joe Bloggs Funding Council",
-  "license": ["MIT", "BSD-3", "GPL-3.0"],
+  "initialise_git_repository": true,
```

Comment:
After a bit of playing it seems that only `True` is accepted in the CLI. So no need to test for `true`, `yes`, `Y`, `1` afaiu.

Reply:
Yeah, I checked that myself. `y` is what you type, or other thing are accepted - but they still evaluate to `True`.